src/viperleed/calc/classes/atom.py

```diff
@@ -12,12 +12,15 @@
 __created__ = '2019-06-13'
 __license__ = 'GPLv3+'
 
+from dataclasses import field
+from typing import Dict
 import copy
 import logging
```

Comment:
It already is?

Reply:
As a string, yes, but we've been sorting on the module name, irrespective of whether it is an `import module` or a `from module import`. I would stick to this, as it makes it easier to group imports. Here it would be `copy`, `dataclasses`, `logging`, `typing`.